app/src/main/java/com/github/dedis/student20_pop/PoPApplication.java

```diff
@@ -203,6 +170,15 @@ public HighLevelProxy getProxy(URI host) {
         }
     }
 
+    /**
+     * Add a new LAO to the app
+     *
+     * @param lao to add
+     */
+    public void createLao(Lao lao) {
+        laos.put(lao.getId(), lao);
+    }
```

Comment:
Should it also add the lao id to the Person's list of LAOs ? if not, where is it done?

Reply:
It is done at Lao creation.

We do not really use the Model Person yet, so it is not really important